src/style/design-tokens/generate-css-variables.util.spec.js

```diff
@@ -0,0 +1,43 @@
+import generateCssVariables from "./generate-css-variables.util";
+import baseTheme from "../themes/base";
+
+describe("generateCssVariables", () => {
+  const tokens = {
+    colorsActionMajor500: "pink",
+    colorsUtility500: "yellow",
+    colorsAction700: "black",
+  };
+  let theme;
+
+  beforeEach(() => {
+    theme = {
+      ...baseTheme,
+      ...tokens,
+    };
+  });
+
+  it("should generate multiline string", () => {
+    const output = generateCssVariables(theme);
```

Comment:
could we move this `const` upper block under the `describe` - this will allow us to delete line 28